language-service/language_service/service/vocab/__init__.py

```diff
@@ -0,0 +1,30 @@
+from .cedict import get_definitions as get_cedict_definitions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in public package](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)